reporter/build.gradle.kts

```diff
@@ -34,6 +34,7 @@ repositories {
 dependencies {
     api(project(":model"))
 
+    implementation(project(":downloader"))
```

Comment:
Would it make sense to move the VcsHost to `model.util` instead?

Reply:
Now that you mention it, I'd rather rename `model.utils` to `model.matchers` as it only contains matchers, or even move the matchers out of the `model` module, as they are not really data model classes...

Reply:
Do we actually have a place where we can put (utility) classes which depend on the model and are shared accross several modules (e.g. analyzer, scanner evaluator or reporter). I think we lack it if I'm not mistaken. If I'm right with this would it make sense to keep those here as other classes with such dependencies might follow?

Reply:
I had a similar thought and am looking into maybe introducing something like a `model-utils` module right; but while doing that I also just discovered another problem with our model API. I'll soon file a PR for a quick fix.

Reply:
For now, please see https://github.com/heremaps/oss-review-toolkit/pull/1917 for the quick fix.